codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/s3/S3Expires.kt

```diff
@@ -0,0 +1,31 @@
+package software.amazon.smithy.aws.swift.codegen.customization.s3
+
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.model.shapes.ServiceShape
+import software.amazon.smithy.model.transform.ModelTransformer
+import software.amazon.smithy.swift.codegen.SwiftSettings
+import software.amazon.smithy.swift.codegen.integration.SwiftIntegration
+import software.amazon.smithy.swift.codegen.model.defaultName
+import software.amazon.smithy.swift.codegen.model.expectShape
+
+class S3Expires : SwiftIntegration {
+
+    override fun enabledForService(model: Model, settings: SwiftSettings): Boolean {
+        return model.expectShape<ServiceShape>(settings.service).isS3
+    }
+
+    override fun preprocessModel(model: Model, settings: SwiftSettings): Model {
+        // Find all the members named "Expires" in all of the output structures
+        // and change their shape from `Timestamp` to `String`
+        val updates = model.structureShapes
+            .filter { it.defaultName().endsWith("Output") }
```

Comment:
So this is indeed the suffix we want to check for, but this currently is failing for tests (although it works as we want when generating the models). All output models are given a `Response` suffix but the timing (and perhaps other factors) are different when executed for tests. I need to dig in to what exactly is going on in our test flow to get tests passing.